go/caddy/pfpki/handlers/handlers.go

```diff
@@ -97,6 +97,42 @@ func GetSetCA(pfpki *types.Handler) http.Handler {
 	})
 }
 
+func ResignCA(pfpki *types.Handler) http.Handler {
+	return http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
+		o := models.NewCAModel(pfpki)
+		var Information types.Info
+		var err error
+
+		Error := types.Errors{Status: 0}
+
+		switch req.Method {
+
+		case "POST":
+			vars := mux.Vars(req)
+			body, err := ioutil.ReadAll(req.Body)
+			if err != nil {
+				Error.Message = err.Error()
+				Error.Status = http.StatusInternalServerError
```

Comment:
Add a break to avoid things continuing 
```suggestion
				Error.Status = http.StatusInternalServerError
				break
```

Reply:
I manage the answer with the manageAnswer function, so i need to continue